File: sdk/python/examples/08_api_keys.py
"""08_api_keys:创建 / 列出 / 撤销(明文仅返回一次)。"""
from helpers import client

combee = client()
created = combee.api_keys.create("production")
print("plaintext key (once):", created["key"])
print("keys:", [k["id"] for k in combee.api_keys.list()])
combee.api_keys.revoke(created["id"])
print("revoked")

File: sdk/typescript/examples/08_api_keys.mjs
// 08_api_keys:创建 / 列出 / 撤销(明文仅返回一次)。
import { client } from "./00_helpers.mjs";
const combee = client();
const created = await combee.apiKeys.create("production");
console.log("plaintext key (once):", created.key);
console.log("keys:", (await combee.apiKeys.list()).map((k) => k.id));
await combee.apiKeys.revoke(created.id);
console.log("revoked");
